app/views/webchat/ConstructionIndustrySchemeView.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
```

Comment:
this view and the associated spec have been recently removed in the main branch, from this ticket: https://jira.tools.tax.service.gov.uk/browse/DEP-3709